docs/en/users/04_Subscriptions.md

```diff
@@ -1,11 +1,90 @@
 # Adding a feed
 
  1. To add a feed, copy the URL of its RSS or Atom file (for instance, the Framablog RSS URL is `https://framablog.org/feed/`). FreshRSS is able to automatically find the address of the feed for websites that are declaring it in a standard way.
- 2. On FreshRSS, click “Subscriptions management”.
- 3. Paste the URL in “Add an RSS feed” below the page title.
+ 2. On FreshRSS, click the plus button next to “Subscriptions management”.
+ 3. Paste the URL in “Feed URL” below the page title.
```

Comment:
You can remove `below the page title` since this has changed. I would rephrase it `Paste the URL in the “Feed URL” field.`